manifests/05_role.yaml

```diff
@@ -66,6 +67,8 @@ rules:
   - apps
   resources:
   - deployments
+  - deployments/finalizers
```

Comment:
Do we really need all of the verbs for these two resources?

verbs:

- get
- create
- delete
- update
- list
- watch

Reply:
This comment applies in all RBAC changes.

Reply:
If you need help testing this please let me know.

Reply:
Have you confirmed that all these RBAC changes are required?

Reply:
Yes, I suppose. Or else, for each of it, there is a log message similar to the above